### PR TITLE
Update screenshots

### PR DESCRIPTION
**Pull request by Conor0Callaghan**
Updating screenshots

Closes: #49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Please make sure you have these dependencies first before building.
 
 ```
 gtk4
-libadwaita-1
+libadwaita-1-0
```
Comment (dusansimic) [inline README.md:25]: Why?

**Review by dusansimic — CHANGES_REQUESTED** [review]
Disable maximize and minimize buttons for screenshots please.

File: README.md (hunk 1 above)
Comment (Conor0Callaghan) [reply README.md:25]: Im not sure on this one @dusansimic , when I was installing and adding the pre-reqs on the Ubuntu dev host I had, libadwaita-1 returned no results, only when searching libadwaita-1-0 , should I revert?

Comment (Conor0Callaghan) [reply README.md:25]: Sounds good no worries :)

**Review by dusansimic — CHANGES_REQUESTED** [review]
Sorry to bother again but the close button seems to be missing (just minimize and maximize needed to be removed, you can do it using GNOME Tweaks app). Also, it looks like the icon pack is a bit different from the Adwaita one which is default for vanilla GNOME. For example menu button has thinner lines and file open icon is different from one on original screenshot. Could this be fixed?

**Comment by Conor0Callaghan** [comment]
> Sorry to bother again but the close button seems to be missing (just minimize and maximize needed to be removed, you can do it using GNOME Tweaks app). Also, it looks like the icon pack is a bit different from the Adwaita one which is default for vanilla GNOME. For example menu button has thinner lines and file open icon is different from one on original screenshot. Could this be fixed?

No worries, I guess this is a Ubuntu thing, Ill take a look :)

**Comment by Conor0Callaghan** [comment]
Apologies @dusansimic that's really messy, i was trying to squash it on the CLI but failed miserably 😢

**Comment by dusansimic** [comment]
No problem 😁! The icons still seem to be wrong.

**Comment by Conor0Callaghan** [comment]
> No problem 😁! The icons still seem to be wrong.

hopefully thats all fixed now, some odd Ubuntu thing forcing the old icon set 🤦‍♂️

**Comment by Conor0Callaghan** [comment]
Happy with this one @dusansimic ?

**Review by dusansimic — APPROVED** [review]

**Comment by dusansimic** [comment]
Looks good! Thanks!

**Pull request merged**